build/CodeSign/certificate.ignore

```diff
@@ -15,4 +15,6 @@ bin\*\*.wixlib, Ignore wix artifact
 bin\*\*.wixpdb, Ignore wix artifact
 bin\*\*.msidir\*.cab, Ignore embedded cab files
 
-binaries-windows-release\*, Ignore downloaded binary artifacts
\ No newline at end of file
+binaries-windows-release\*, Ignore downloaded binary artifacts
```

Comment:
I think you can revert all of the exclusions if my XDT extensions signing PR is completed.